### ISSUE TITLE
how to integrate into vllm

### ISSUE THREAD

**Issue by FWXT** [smpanaro/token-recycling#1]
hi, is there easy way to integrate it into vllm.  thx!

**Comment by smpanaro** [comment]
Hey. I haven’t tried this with vllm. I haven’t thought about this project in a while, but if I remember correctly it’s mostly about pre and post processing tokens and logits, so I assume that integrating it would be possible.
